fizz_buzz/g3wanghc/fizz_buzz_aye_lemayo.py

```diff
@@ -0,0 +1,14 @@
+def main():
+    print ([fizz_buzz(n) for n in range(1, 100)])
+
+def fizz_buzz(n):
+	if (n % 15 == 0):
+		return 'FizzBuzz'
```

Comment:
Should probably avoid mixing tabs and spaces. Also your tabs are quite large. I think PEP8 would suggest these tabs should be replaced with 4 spaces